pom.xml

```diff
@@ -159,6 +160,11 @@ SOFTWARE.
       <artifactId>reactor-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
@iColdLight sort your dependencies, compile/provided scope first,
and only then test one